model_analyzer/triton/server/server_local.py

```diff
@@ -62,13 +62,11 @@ def start(self):
         if self._server_path:
             # Create command list and run subprocess
             cmd = [self._server_path]
-            cmd += self._server_config.to_cli_string().replace('=',
-                                                               ' ').split()
+            cmd += self._server_config.to_cli_string().replace('=', ' ').split()
+            # List GPUs to be used by tritonserver
             triton_env = os.environ.copy()
-            if len(self._gpus) >= 1 and self._gpus[0] != 'all':
-                visible_gpus = GPUDeviceFactory.get_cuda_visible_gpus()
-                triton_env['CUDA_VISIBLE_DEVICES'] = ','.join(
```

Comment:
DId you find this to be redundant?

Reply:
You mean if it includes all GPUs, then just don't bother setting the flag?

Reply:
Im just asking why it was removed.

Reply:
It is moved to `server_factory.py`.